.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: 🐛 Create a a bug report.
+labels: bug, needs triage
```

Comment:
👏 I've create the label [`needs triage`](https://github.com/scanapi/scanapi/labels/needs%20triage) for us!